src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/jvm/JvmGenerationTuningPolicy.java

```diff
@@ -51,20 +55,17 @@
       OLD_GEN_HEAP_USAGE
   );
   private static final long COOLOFF_PERIOD_IN_MILLIS = 48L * 24L * 60L * 60L * 1000L;
+  private static final String UNDERSIZED_DATA_FILE_PATH = "/tmp/JvmGenerationTuningPolicy_Undersized";
+  private static final String OVERSIZED_DATA_FILE_PATH = "/tmp/JvmGenerationTuningPolicy_Oversized";
```

Comment:
nit: do we want these in `/tmp`? Is there another config or data folder that we can use?